receiver/k8sobjectsreceiver/testdata/e2e/collector/configmap.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Name }}-config
+  namespace: default
+data:
+  relay: |
+    exporters:
+      otlp:
+        endpoint: {{ .HostEndpoint }}:4317
+        tls:
+          insecure: true
+    extensions:
+      health_check: {}
+    processors:
+      transform/k8sobjectsNS:
```

Comment:
Made the choice to use transform processor instead of doing the sanitization in the test, with the idea that users can refer to the config and the logs generated in testdata to make some sense of what to expect when working with data from this receiver.